README.md

```diff
@@ -15,6 +15,65 @@ To see what we are working on acquaint yourself with our:
 
 To contribute, join the team and we will get you up and running, or, show us your moxie and submit a pull request!
 
+# How to setup for development?
+You need to clone this repository and 2 others (database, asset).
+
+The PerpetuumServer handle server communication with the client.
+
+The OPResource is an asset host that the client access when connecting to a server, the address can be configured in the data folder in the ini file.
+
+The OPDB is contains the database schema definition and the migration patches for each release.
+
+## Getting the source and the necessary files
+1. Clone this repository (github page: [here](https://github.com/OpenPerpetuum/PerpetuumServer)) (folder/PerpetuumDedicated)
+2. Clone the OPDB repository (github page: [here](https://github.com/OpenPerpetuum/OPDB)) (folder/OPDB)
+3. Clone the asset respo (github page: [here](https://github.com/OpenPerpetuum/OPResource)) (folder/OPResource)
+4. Get official Perpetuum Dedicated server installer, depot ID (with steamcmd): **693061**
+    - (steam) Library -> Home -> Above search select *Tools*, search for *PerpetuumDedicated*, install it
+5. Get layers archive and icons, available in a public google drive folder [here](https://drive.google.com/drive/folders/1V6Dnytm0eZCuA_2PZHeUGslNzPrmXKtD)
+Note that the Gamma_layers is compatible with release P26+
+
+Here save the layers rar file + icons zip file.
+
+6. (Requires Windows for now) Run the PerpetuumDedicated from steam or directly the exe, install SQL server express (known issues with Win11+, unable to install, use windows 10 to install), initialize the DB when asked at the end.
+
+At this point, take note of the path of the installed server, usually `C:\PerpetuumDedicated`.
+
+## Move/extraction of files
+1. Extract layers archive into the data folder of the PerpetuumDedicated installation
+2. Extract icons from the archive to the `OPResource/lang0000` folder.
+
+Note that the icons archive does not contain icons for new items, the resource server will substitute the default icon instead. The default icon is the blue cube.
```

Comment:
The default icon for new OPP icons is [this](https://github.com/OpenPerpetuum/OPResource/blob/dev/lang0000/icons/default/00000000.png) one.

Reply:
Maybe just remove the last sentence.

Reply:
Oh I thought it was the blue cube, ok, updating the readme.